src/components/Layout.tsx

```diff
@@ -161,27 +129,53 @@ const Layout: React.FC<IProps> = ({
                 isPageRightToLeft={isPageRightToLeft}
                 originalPagePath={pageContext.originalPath!}
               />
-              <ContentContainer>
+
+              <Flex
+                position="relative"
+                margin="0px auto"
+                minHeight="100vh"
+                flexFlow="column"
+                sx={{
+                  "@media": {
+                    minWidth: "1024px",
+                    maxWidth: "1504px",
+                  },
+                }}
```

Comment:
Not quite sure what happened here. 🤔 
```suggestion
                maxW={{ 
                  lg: lightTheme.variables.maxPageWidth 
                }}
```
And use the existing `lightTheme` import, because `maxPageWidth` is currently not applied in the custom Chakra theme yet.

Reply:
Correct. Lets do this ^

Reply:
okay 👍🏼